src/session.rs

```diff
@@ -648,9 +654,10 @@ where
             .map(Self::map_handshake_response)
             .await
         {
-            Ok(_) => Ok(Client {
+            Ok(NewSessionResponse { capabilities, .. }) => Ok(Client {
                 tx,
                 is_legacy: false,
+                remote_capabilities: capabilities.as_object().unwrap().clone(),
```

Comment:
Seeing this, should we perhaps just store (and expose) the entire `NewSessionResponse` rather than _only_ the capabilities? There's not much else in there at the moment, but the session id may be handy, and who knows if more fields may be added in the future?